CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 For 1.x changes, go [here](https://github.com/twilio/twilio-video.js/blob/support-1.x/CHANGELOG.md).
 
+2.0.0-beta8 (in progress)
```

Comment:
2.0.0-beta7 is still in progress. So this wont be part of that I guess?

Reply:
@syerrapragada Yes. We will release this after releasing beta7.